tarsier/tag_utils.ts

```diff
@@ -386,3 +388,36 @@ const showMapElements = () => {
     (element as HTMLElement).style.opacity = element.getAttribute('data-original-opacity') || '1';
   });
 }
+
+window.hideElements = () => {
+  const allElements = document.body.querySelectorAll("*");
+  allElements.forEach((el) => {
+    const element = el as HTMLElement;
+
+    // if the element has the visibility property, then copy the value of that property into
+    // a new attribute called 'reworkd-original-visibility'
+    if (element.style.visibility){
+      element.setAttribute('reworkd-original-visibility', element.style.visibility);
```

Comment:
let's store this string in a variable so that we can reference it in both places